src/javascript/js-webshim/minified/polyfiller.init.js

```diff
@@ -0,0 +1,6 @@
+(function($) {
+    $.webshim.bugs.bustedValidity = !('minLength' in document.createElement('input'));
+    $.webshims.setOptions('basePath', Zikula.Config.baseURL+'javascript/js-webshim/minified/shims/'); 
+    $.webshims.activeLang(Zikula.Config.lang);
+    $.webshims.polyfill(Zikula.Config.polyfillFeatures);
+})(jQuery);
```

Comment:
It's using "System::isDevelopmentMode()", in the same manner in which other libraries are loaded, based on that condition.